src/Builder/ClassModifier/ClassInstance/Wire/WireClassInstanceModifier.php

```diff
@@ -75,8 +75,18 @@ public function wire(
         $methodNode = $this->classNodeFinder->findMethodNode($classInformationTransfer, $targetMethodName);
         if (!$methodNode) {
             $classInformationTransfer = $this->commonClassModifier->overrideMethodFromParent($classInformationTransfer, $targetMethodName);
-            $methodNode = $this->classNodeFinder->findMethodNode($classInformationTransfer, $targetMethodName);
         }
+        $methodNode = $this->classNodeFinder->findMethodNode($classInformationTransfer, $targetMethodName);
```

Comment:
The behavior here looks strange. You always rewrite here the $methodNode from line 75 even if the `$classInformationTransfer` is the same. 

And after that you rewrite it again in line 89 even if the `$classInformationTransfer` is the same.

Reply:
Thanks 
created the method `getMethodNode`